src/charts/donut/donut-pct-chart-directive.js

```diff
@@ -305,7 +307,7 @@ angular.module('patternfly.charts').directive('pfDonutPctChart', function (c3Cha
 
           // default to 'used' info.
           centerLabelText = { bigText: $scope.data.used,
-                              smText:  $scope.config.units + ' Used' };
+                              smText:  $i18next('pfDonutPctChart.used', { units: $scope.config.units }) };
```

Comment:
Why not just use the words themselves as the keys in the hash?  Is the thought that people will want to alter these individually per component?